devs/docs/tests.rst

```diff
@@ -10,6 +10,28 @@ Run the all of `doctests`_::
 
     $ ./gradlew itest
 
+For running itest on Windows, `WSL needs to be installed`_. If the downloaded
+Linux distro comes without Java, you can install it by running the following
+commands::
+
+    $ sudo add-apt-repository ppa:openjdk-r/ppa
+    $ sudo apt-get update
+    $ sudo apt install openjdk-11-jdk.
```

Comment:
It looks like a trailing dot slipped in here.

Reply:
Fixed, thanks